### ISSUE TITLE
Better way for client programs to do a HEAD request to a collection endpoint

### ISSUE THREAD

**Issue by elelay** [Kinto/kinto-http.js#128]
If they want to get the record count in a collection, for instance.

Here is current way of doing it (getting the api from a kinto.js collection):

```
var api = tickets.api.bucket(tickets.bucket).collection(tickets.name);
var headers = api._collOptions({});
headers["Authorization"] = "Basic " + btoa("token:tr0ub4d@ur");
api.client.execute({ path: "/buckets/default/collections/oldTickets/records", method:"HEAD", headers}, {raw:true})
```

**Comment by Natim** [comment]
I had a similar problem to get the ETag header value.
Maybe we should add a way to ease headers access.

**Comment by dstaley** [comment]
@leplatrem was this implemented with #425?

**Comment by leplatrem** [comment]
Yes! Good catch :)
